src/service/components/mpris.js

```diff
@@ -325,7 +325,7 @@ var Player = GObject.registerClass({
 
     get LoopStatus() {
         if (this._LoopStatus === undefined)
-            this._LoopStatus = 'None';
+            this._LoopStatus = null;
```

Comment:
`null` is an invalid type for this property; what's the purpose of doing this?

Reply:
These fields are optional thus I need some way to know whether given player sets them or not. See: https://specifications.freedesktop.org/mpris-spec/2.2/Player_Interface.html

If you wish for me to use different fallback value than `null`, let me know.

Reply:
I think the fallbacks of `LoopStatus`/`None` and `Shuffle`/`false` would best.

I reason that if the MPRIS player doesn't support the properties the `try..catch` will return the fallback silently, and if the remote doesn't then it should just ignore those fields. If one does and the other doesn't, I think a player that doesn't support `LoopStatus` is effectively `None` and same for `Shuffle` being `false`.

Does that seem reasonable?

Reply:
Uh but then how would we know if it's good to add these values to requestNowPlaying response so that KDE Connect app can hide not supported controls?

Reply:
Ah I see why you did that now. In that case you can re-use the `_get()` function and leave out the fallback as you did below in the `PlayerRemote` class, but the property getters here in the `Player` class can not return `null`.

The problem is the `Player` class is the base class for both local players we consume and remote players we export. If we return `null` as an exported player our code will throw when we try to pack the `null` into a DBus variant, while the other side of DBus will probably get an incorrect error code (or no reply at all).

Unfortunately, when we're exporting players we sort of have to pretend to support everything, because we can't really dynamically export different parts of the MPRIS interface (not easily anyways).

Reply:
Sidenote: If you do want to go through to work of supporting optional properties on MPRIS proxies we export, you can open another PR and I'll try and help you through that.

Reply:
Ok, since kde-connect-android MR just got merged I fpushed this PR according to your suggestions.